tests/test-definition.yaml

```
@@ -7,6 +7,8 @@ dimensions:
   - name: hive
     values:
       - 2.3.9-stackable0.4.0
+      - 2.3.9-stackable0.4.0
```

Sorry, fat fingers :see_no_evil: 
```suggestion
      - 2.3.9-stackable0.5.0
```